### ISSUE TITLE
dexecutor crashes on db.connect(), Deno 1.0.4 & Postgres

### ISSUE THREAD

**Issue by Ryan-Haig12** [denjucks/dexecutor#5]
I was using deno 1.0.2 with dexecutor just fine for a few days now. I performed all CRUD operations with ease thanks to dexecutor. However, I just updated deno to 1.0.4 and my app now crashes when I call `await db.connect()` exactly as advised in the project's README

# error
```
% denon run --allow-net --unstable server.ts
[denon] v2.0.2
[denon] watching path(s): *.*
[denon] watching extensions: ts,js,json
[denon] starting `deno run --allow-net --unstable server.ts`
Compile file:///Users/ryanhaig/Documents/Programming/calorie-counter/calorie-counter-deno/server.ts
thread 'main' panicked at 'already borrowed: BorrowMutError', /private/tmp/rust-20200509-70766-vc2112/rustc-1.43.1-src/src/libcore/cell.rs:878:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
[denon] app crashed - waiting for file changes before starting ...
```

# server.ts
```typescript
import { Application } from 'https://deno.land/x/oak/mod.ts'
import db from './db/config.ts'
import router from './routes.ts'
import querySanitizer from './utils/querySanitizer.ts'
const app = new Application()
await db.connect() // <- this is the error
app.use(querySanitizer)
app.use(router.routes())
app.use(router.allowedMethods())
const port = 4000
console.log(`🦕 Deno 🦕 Server running on port 🦖 ${ port } 🦖`)
await app.listen({ port })
```

# ./db/config.ts
```typescript
import Dexecutor from "https://deno.land/x/dexecutor/mod.ts"
const client = "postgres"
// Creating the query executor
let db = new Dexecutor({
    client: client,
    connection: {
        // correct connection data
    }
})
export default db
```

**Comment by Ryan-Haig12** [comment]
Downgrading to deno 1.0.3 seemed to fix this.

`https://github.com/denoland/deno/releases/tag/v1.0.4`
upgraded Rust crates maybe?

**Comment by denjucks** [comment]
There seems to be an issue in 1.0.4 that the deno team is working on that was causing the issue. For now you'll have to downgrade to an earlier version. Gonna keep this open for now.

**Comment by denjucks** [comment]
Should be fixed now for Deno v1.1.0
